### PR TITLE
Williamhe/add php redis

### PR DESCRIPTION
**Pull request by william-msft**
- [ ] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as \#\<issue\_number\>.
- [ ] Tests are included and/or updated for code changes.
- [ ] Proper license headers are included in each file.

### CODE REVIEW

File: images/runtime/php-fpm/7.2/base.stretch.Dockerfile [hunk 1]
```diff
@@ -70,6 +70,7 @@ RUN docker-php-ext-configure pdo_odbc --with-pdo-odbc=unixODBC,/usr \
         xsl \
     && pecl install imagick && docker-php-ext-enable imagick
 
+RUN pecl install redis && docker-php-ext-enable redis
```
Comment (arroyc) [inline images/runtime/php-fpm/7.2/base.stretch.Dockerfile:73]: remove

File: images/runtime/php/8.0/base.buster.Dockerfile [hunk 2]
```diff
@@ -71,6 +71,8 @@ RUN docker-php-ext-configure pdo_odbc --with-pdo-odbc=unixODBC,/usr \
 #       xmlrpc \
         xsl
 
+RUN pecl install redis && docker-php-ext-enable redis
```
Comment (arroyc) [inline images/runtime/php/8.0/base.buster.Dockerfile:74]: remove

File: images/runtime/php/template.base.Dockerfile [hunk 3]
```diff
@@ -71,6 +71,12 @@ RUN docker-php-ext-configure pdo_odbc --with-pdo-odbc=unixODBC,/usr \
 #       xmlrpc \
         xsl
 
+# https://github.com/Imagick/imagick/issues/331
```
Comment (arroyc) [inline images/runtime/php/template.base.Dockerfile:74]: remove the comment, it was just a copy paste

**Review by arroyc — CHANGES_REQUESTED** [review]
add unit tests for checking the redis module is installed and enabled in php runtime images

nit: pr header make it more clear and well described

File: tests/Oryx.RuntimeImage.Tests/Php-fpm/PhpFpmImageTest.cs [hunk 4]
```diff
@@ -149,5 +149,32 @@ public void PhpFpmRuntimeImage_Contains_VersionAndCommit_Information(string vers
...
                 result.GetDebugInfo());
         }
+
+        [Theory]
+        [InlineData("8.0")]
+        [InlineData("7.4")]
+        [InlineData("7.3")]
+        [InlineData("7.2")]
+        [InlineData("7.0")]
+        public void Redis_IsInstalled(string imageTag)
```
Comment (arroyc) [inline tests/Oryx.RuntimeImage.Tests/Php-fpm/PhpFpmImageTest.cs:159]: add some of these tests in phpfpm as well as we have added redis for both fpm based image and apache baded image

**Comment by arroyc** [comment]
change the pr name to be something like "add support for redis packages for php runtime images"

File: tests/Oryx.RuntimeImage.Tests/Php-fpm/PhpFpmImageTest.cs (hunk 4 above)
Comment (william-msft) [reply tests/Oryx.RuntimeImage.Tests/Php-fpm/PhpFpmImageTest.cs:159]: unit tests have been added now

**Comment by william-msft** [comment]
Created a new MR with better branch name. Going to close this duplicate MR

**Pull request closed without merging**